File: services/message_storage/purge_inbox.py
import requests

headers = {'Mailsac-Key': 'YOUR_API_KEY_HERE'}
url = 'https://mailsac.com/api/addresses/bclinton@mailsac.com/messages'
r = requests.delete(url, headers=headers)
print(r.status_code)

'''
204
'''

File: services/message_storage/purge_inbox.js
const superagent = require('superagent')
const mailsac_api_key = 'YOUR_API_KEY_HERE'

superagent
  .delete('https://mailsac.com/api/addresses/bclinton@mailsac.com/messages/')
  .set('Mailsac-Key', mailsac_api_key)
  .then((res) => {
      console.log(res.status)
  })
  .catch(err => {
      console.error(err.message);
  })

/*
204
*/
